src/vcpkg/commands.version.cpp

```diff
@@ -6,17 +6,26 @@
 #include <vcpkg/vcpkgcmdarguments.h>
 #include <vcpkg/vcpkgpaths.h>
 
-namespace vcpkg::Commands::Version
+namespace
 {
-    const char* version() noexcept
-    {
-        return VCPKG_BASE_VERSION_AS_STRING "-" VCPKG_VERSION_AS_STRING
+    using namespace vcpkg;
+
+    DECLARE_AND_REGISTER_MESSAGE(
+        VersionCommandHeader,
+        (msg::version),
+        "",
+        "Vcpkg package management program version {version}\n\nSee LICENSE.txt for license information.");
```

Comment:
```suggestion
        "vcpkg package management program version {version}\n\nSee LICENSE.txt for license information.");
```

nit, but it's spelled `vcpkg`; close as wontfix if you don't want to change behaviour